03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+path=$(pwd)
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+    echo "Script to search for files with the same md5  checksum"
+    echo "Usage: sript [OPTION] or [PATH]"
```

Comment:
Change sript to find_dup_files.sh or to $0

Reply:
Done